### PR TITLE
[ENG-44064] Attendees form

### PR DESCRIPTION
**Pull request by quangtran-sked**

### CODE REVIEW

File: forms/attendees/mex_definition/ui_def.json [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+        "placeholder": "form.AttendeeListPage.SearchPlaceholder",
+        "filterOnProperties": ["Contact.FullName"]
+      },
+      "itemClickDestination": "AttendeeDetailPage"
+    },
+    "AttendeeDetailPage": {
+      "type": "flat",
+      "events": {
+        "onPreDataLoad": "cf.filterAlertsByClientId(formData.Alerts, pageData)",
+        "onPreDataSave": "cf.remoteAlertsByClientId(pageData)"
```
Comment (cuong-duong) [inline forms/attendees/mex_definition/ui_def.json:60]:
```suggestion
        "onPreDataSave": "cf.removeAlertsByClientId(pageData)"
```

File: forms/attendees/mex_definition/ui_def.json [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+              "items": [
+                {
+                  "text": "call",
+                  "theme": "default",
+                  "icon": "phone",
+                  "behavior": {
+                    "type": "phone",
+                    "phoneNumberExpression": "pageData.Contact.Phone"
+                  }
+
```
Comment (cuong-duong) [inline forms/attendees/mex_definition/ui_def.json:82]:
```suggestion
```

File: forms/attendees/custom_functions/custom-function/src/index.frontend.ts [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+}
+
+function filterAlertsByClientId(alerts, pageData) {
+  return {
+    Alerts: alerts.filter((element) => element.ClientId === pageData.ContactId),
+    ...pageData
+  };
+}
+
+function remoteAlertsByClientId(pageData) {
```
Comment (cuong-duong) [inline forms/attendees/custom_functions/custom-function/src/index.frontend.ts:24]:
```suggestion
function removeAlertsByClientId(pageData) {
```

File: forms/attendees/mex_definition/ui_def.json (hunk 1 above)
Comment (cuong-duong) [inline forms/attendees/mex_definition/ui_def.json:60]:
```suggestion
        "onPreDataSave": "cf.removeAlertsByClientId(pageData)"
```

File: forms/attendees/mex_definition/ui_def.json (hunk 2 above)
Comment (quangtran-sked) [reply forms/attendees/mex_definition/ui_def.json]: Solved

File: forms/attendees/custom_functions/custom-function/src/index.frontend.ts (hunk 3 above)
Comment (quangtran-sked) [reply forms/attendees/custom_functions/custom-function/src/index.frontend.ts]: Solved

File: forms/attendees/mex_definition/ui_def.json (hunk 1 above)
Comment (quangtran-sked) [reply forms/attendees/mex_definition/ui_def.json]: Solved

**Review by cuong-duong — APPROVED** [review]

**Pull request merged**